react-app/src/components/Result.js

```diff
@@ -36,34 +37,8 @@ function Result(props) {
     return total;
   }
 
-  function getLabel() {
-    // attempt to get MRN and name with fhirpath
-    const patient = fhirpath.evaluate(props.bundle, "Bundle.descendants().resource.where(resourceType='Patient')")[0];
-    if (patient) {
-      const mrn = patient.id;
-      const name = patient.name[0].text;
-
-      const isMasked = fhirpath.evaluate(patient, 'Patient.identifier.extension.valueCode')[0] === 'masked';
-
-      // if both MRN and name -- return string w / both
-      if (typeof mrn === 'string' && mrn.length > 0 && typeof name === 'string' && name.length > 0 && !isMasked) {
-        const label = mrn.concat(': ').concat(name);
-        return <p className="accordion-result-label">{label}</p>;
-      }
-      // if either MRN or name -- return string / the available one
-      if (typeof name === 'string' && name.length > 0) {
-        return <p className="accordion-result-label">{name}</p>;
-      }
-      // if neither MRN nor name -- "Patient " + patient_resource_id
-      if (typeof mrn === 'string' && mrn.length > 0) {
-        return <p className="accordion-result-label">{mrn}</p>;
-      }
-    }
-
-    // if no patient resource ID -- return "Patient " + props.id
-    let label = 'Patient';
-    label = label.concat(' ').concat(props.id.toString());
-    return <p className="accordion-result-label">{label}</p>;
+  function getLabelJSX() {
+    return <p className="accordion-result-label">{getLabel(props.bundle, props.id)}</p>;
```

Comment:
Super small, but in updating this to be a paragraph tag, we now have margin-bottom on the element. Could we just add margin-bottom: 0 to the accordion result label class